tests/tuning/tuners/test_gaussian_process.py

```diff
@@ -20,3 +20,19 @@ def test___init__(self):
 
         # assert
         assert instance._MODEL_CLASS == GaussianProcessRegressor
+        assert instance._metamodel_kwargs == {'length_scale': 0.1}
+
+
+class TestGaussianProcessExpectedImprovementTuner(TestCase):
+    """Test GaussianProcessExpectedImprovementTuner class."""
+
+    def test___init__(self):
+        # setup
+        instance = MagicMock()
+
+        # run
+        instance = GPEiTuner(MagicMock())
+
+        # assert
+        assert instance._MODEL_CLASS == GaussianProcessRegressor
```

Comment:
I would think one should check identity rather than equality for classes, i.e. `assert instance._MODEL_CLASS is GaussianProcessRegressor`. (Even though I believe the semantics of the two checks are the same in this case.)